tests/test_cloudformation/test_cloudformation_stack_crud.py

```diff
@@ -568,6 +568,9 @@ def test_describe_stack_events_shows_create_update_and_delete():
     except StopIteration:
         assert False, "Too many stack events"
 
+    with pytest.raises(BotoServerError):
```

Comment:
Hi @modamod, would you mind adding some assertions against this error?
```
with pytest.raises(..) as e:
    ...
e.value.response["Error"]["Message"].should.equal(..)
```
(Same for the other two error-scenarios in this PR)

Reply:
Not at all, I will add some assertions to all the errors I added.

Reply:
I added some assertions against the exceptions in tests. please let me know if you need me to add anything else?